lib/shared/robot/index.js

```diff
@@ -215,3 +215,17 @@ exports.printError = (error) => {
 exports.printMessage = (message, data) => {
   console.log(exports.buildMessage(message, data));
 };
+
+/**
+ * @summary Log a message to the host's console
+ * @function
+ * @public
+ *
+ * @param {Object} [data] - data
```

Comment:
It does have to be an object, otherwise `buildMessage()` will yell (see line 58: https://github.com/resin-io/etcher/pull/1212/files#diff-0ccd387a7e7d9c117af87e3bdbd6a9ebR58)